protocol/src/main/protobuf/Transactor.proto

```diff
@@ -0,0 +1,27 @@
+syntax = "proto3";
+package protocol;
+option java_package = "io.mediachain.protocol.transactor";
+
+message MultihashReference {
+    string reference = 1;
+}
+
+message InsertRequest {
+    bytes canonicalCbor = 1;
+}
+
+message UpdateRequest {
+    bytes chainCellCbor = 1;
+}
+
+service TransactorService {
+    rpc FetchObjectChainHead(MultihashReference) returns
+        (MultihashReference);
+
+    rpc InsertCanonicalRecord(InsertRequest) returns
+        (MultihashReference);
+
+    rpc UpdateCanonicalRecord(UpdateRequest) returns
```

Comment:
this method name is not exactly right -- we are not updating the canonical record, just its chain pointer.